examples/README.md

```diff
@@ -2,17 +2,59 @@
 
 Here we have put together a hand full of python examples to help get you started. As always feel free to [contact us](https://lob.com/support) directly if you have any questions on implementation.
 
+## Getting started
+Before running these examples make sure to do the following:
 
-## csv_address_verification
+1. Install the lob package with pip or easy_install:
```

Comment:
It is pointing to the master copy. I was wrong yesterday.
They do still need to install the package once to use 'import lob'.